pyproject.toml

```diff
@@ -34,10 +35,11 @@ numpy = "^1"
 pytz = "^2024.1"
 
 [tool.poetry.group.dev.dependencies]
-black = "^24.2.0"
+black = "24.2.0"
 pre-commit = "^3.7.1"
-
+tox = "^4.15.1"
+pytest = "^8.2.2"
 
 [build-system]
-requires = ["poetry-core"]
+requires = ["poetry-core>=1.0.0"]
```

Comment:
From [_Is tox supported?_](https://python-poetry.org/docs/faq#is-tox-supported) in Poetry's official docs.